### PR TITLE
Allowing configuration of mongo collection names with a prefixes

### PR DESCRIPTION
**Pull request by marcusdroy**
**New**

- Allows prefixing of of collection names through new spring boot property 'workflow.mongo.collection.prefix'

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -117,3 +117,6 @@ workflow.service.host=localhost:${server.port}
 spring.aop.proxy-target-class=true
 
 logging.level.net.boomerangplatform.service.refactor=DEBUG
+
+workflow.mongo.collection.prefix=cicd
```
Comment (tlawrie) [inline src/main/resources/application.properties:121]: should this default to flow, and then we adjust to CICD?

**Comment by tlawrie** [comment]
QQ for you. How does `#{@mongoConfiguration.prefix(` map to `workflow.mongo.collection.prefix`

**Comment by marcusdroy** [comment]
> QQ for you. How does `#{@mongoConfiguration.prefix(` map to `workflow.mongo.collection.prefix`

MongoConfiguration component prefix method will get called to produce a new collection name with the prefix.

File: src/main/resources/application.properties (hunk 1 above)
Comment (marcusdroy) [reply src/main/resources/application.properties:121]: I did intend this to be blank instead of cicd.

**Pull request merged**